command-line.txt

```diff
@@ -1,11 +1,11 @@
 What is the command that displays your current working directory?
-your_answer = ____________
+your_answer = pwd
 
 What is the command to list all of the files and directories in your current working directory?
-your_answer = ____________
+your_answer = ls
 
 What is the command to move the image of the dirty puppy into the bath tub?
-your_answer = _____________
+your_answer = mv ~/workspace/Sherwood-Caballero/command-line-proficiency-2015/dirty_puppy.jpg Sherwood-Caballero/bathtub
```

Comment:
assume that you're in the repository directory already. This works, but is unnecessarily precise.